_posts/2017-02-27-protocol-buffers.md

```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
 distribution (described below, in "Implicit Dependencies and Proto Toolchains").
 The following satisfies these dependencies:
 
-> TIP: This is a shortened version of
-> [https://github.com/cgrushko/proto_library/blob/master/WORKSPACE](https://github.com/cgrushko/proto_library/blob/master/WORKSPACE)
+> TIP: Clone https://github.com/cgrushko/proto_library/ to try protobufs in Bazel now.
```

Comment:
Let's linkify the github url

Reply:
GitHub automatically linkifies links - see preview in https://github.com/bazelbuild/bazel-blog/blob/80146308bdfef262d2b1d7102de75af03b6e5992/_posts/2017-02-27-protocol-buffers.md

Reply:
This will be converted to HTML for the bazel blog, so we'll need the []() syntax

Reply:
Good point, done.